### PR TITLE
test 2

### PR DESCRIPTION
**Pull request by jelaniwoods**

### CODE REVIEW

File: app/controllers/producer_sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        session.store(:producer_id, producer.id)
+      
+        redirect_to("/", { :notice => "Signed in successfully." })
+      end
+    else
+      redirect_to("/producer_sign_in", { :alert => "There's no producer account with that email address." })
+    end
+  end
+
+  def remove_cookies
```
Comment (jelaniwoods) [inline app/controllers/producer_sessions_controller.rb:28]: This action name needs to match the one specified in the route, which is currently called `destroy_cookies`.

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,50 @@
...
+
+  # ------------------------------
+
+  # SIGN IN FORM
+  get("/producer_sign_in", { :controller => "producer_sessions", :action => "new_session_form" })
+  # AUTHENTICATE AND STORE COOKIE
+  post("/producer_verify_credentials", { :controller => "producer_sessions", :action => "create_cookie" })
+  
+  # SIGN OUT        
+  get("/producer_sign_out", { :controller => "producer_sessions", :action => "destroy_cookies" })
```
Comment (jelaniwoods) [inline config/routes.rb:25]: The action needs to match the action in the `producer_sessions` controller.

**Comment by jelaniwoods** [comment]
Minor changes, looks good!

**Comment by raghubetina** [comment]
@jelaniwoods Addressed this round of feedback. Thank you

**Pull request closed without merging**